### DISCUSSION TITLE
remote logging to an s3 compatible storage

### DISCUSSION THREAD

**Discussion by alonahmias** [apache/airflow#30307] [Q&A]
i have been trying to setup remote logging for airflow to an s3 compatible storage, and i have set the environment variable right, because i see that i can use it from the code, but when i run a task, it says that this connection can not be found, as if the remote logging feature does not support the usage of environment variables feature.
any one had any luck with it? because it seems that i cant do it, and i can't find what i'm doing wrong.

**Comment by potiuk** [comment]
You needd to check if your connection is properly configured. It's impossible to help you without knowing the details that you know - check for typos, eventually debug the code that retrieves the connection to see if you properly assumed the configuraiton of yours - that would be my advice.
